java/semantickernel-extensions-parent/actionplanner-extension/src/main/java/com/microsoft/semantickernel/planner/actionplanner/ActionPlanner.java

```diff
@@ -36,6 +40,8 @@
 public class ActionPlanner {
     private static final Logger LOGGER = LoggerFactory.getLogger(ActionPlanner.class);
 
+    private static final Pattern CLEAN_PLAN =
```

Comment:
Would be helpful to have comments on patterns. It looks to me like this trying to get at the inner braces in something like {{$input}} (group 1 would be "{$input}"). But when I play with it in jshell, group one is "{{$input}}".